packages/runtime/datastore-definitions/src/dataStoreRuntime.ts

```diff
@@ -140,4 +144,11 @@ export interface IFluidDataStoreRuntime extends
      * Errors raised by distributed data structures
      */
     raiseContainerWarning(warning: ContainerWarning): void;
+
+    /**
+     * NOTE: Search blob/extra concept
```

Comment:
I'd remove "NOTE" and just describe how feature works (in general), and then maybe mention search as an example of where it might be useful

Reply:
Will do! - didn't pay much attention to the comments here since this was just a first attempt.